docs/develop/build-and-run/aot.md

```diff
@@ -23,6 +23,30 @@ USAGE
 ...
 ```
 
+## Output Format: Universal WASM
+
+By default, the `wasmedgec` AOT compiler tool could wrap the AOT-compiled native binary into a custom section in the origin WASM file. We call this the universal WASM binary format.
+
+This AOT-compiled WASM file is compatible with any WebAssembly runtime. However, when this WASM file is executed by the WasmEdge runtime, WasmEdge will extract the native binary from the custom section and execute it in AOT mode.
+
+> Note: On MacOS platforms, the universal WASM format will `bus error` in execution. It's because the `wasmedgec` tool optimizes the WASM in `O2` level by default. We are trying to fix this issue. For working around, please use the shared library output format instead.
```

Comment:
```suggestion
:::note 
On MacOS platforms, the universal WASM format will `bus error` in execution. It's because the `wasmedgec` tool optimizes the WASM in `O2` level by default. We are trying to fix this issue. For working around, please use the shared library output format instead.
:::
```

Reply:
@alabulei1 How about if we use the admonitions according to https://docusaurus.io/docs/markdown-features/admonitions for the note here?

Reply:
Yup, that should be a better way to look at it!

Creating a alert box!